### PR TITLE
[DOCS-5247] Session replay for post-mortems

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Adds a new guide to the RUM guides under the Session Replay section focused on how the Session Replay + Notebooks integration is key to solving post-mortems.

### Motivation
[DOCS-5247](https://datadoghq.atlassian.net/browse/DOCS-5247)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5247]: https://datadoghq.atlassian.net/browse/DOCS-5247?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jamiemilstein — APPROVED** [review]

**Comment by jamiemilstein** [comment]
hey there! wanted to check on this to see if it's all good to merge?

**Comment by rtrieu** [comment]
@jamiemilstein It's up to someone on our docs team to be picked up for review, but I will try to see if they can review it sooner.

File: content/en/real_user_monitoring/guide/using-session-replay-as-a-key-tool-in-post-mortems.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Using Session Replay As A Key Tool In Post-Mortems
+kind: guide
+further_reading:
+- link: 'https://www.datadoghq.com/blog/analyze-user-experience-frustration-signals-with-rum/'
+  tag: 'blog'
+  text: 'Detect user pain points with Datadog Frustration Signals'
```
Comment (estherk15) [inline content/en/real_user_monitoring/guide/using-session-replay-as-a-key-tool-in-post-mortems.md:7]: Doublecheck the spacing, but suggestion to add Notebooks docs to further reading!
```suggestion
  text: 'Detect user pain points with Datadog Frustration Signals'
- link: 'https://docs.datadoghq.com/notebooks/'
  tag: 'documentation'
  text: 'Notebooks'
```

**Review by estherk15 — APPROVED** [review]
Looks good! Added some suggestions, but they're not blockers to this moving forward.

**Comment by rtrieu** [comment]
@estherk15 Implemented your suggestions - thanks!

**Pull request merged**